cv_next/components/ui/dropdownCover.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import { useState } from "react";
+
+type DropdownProps = {
+  title: string;
+  children: React.ReactNode;
+};
+
+const DropdownCover: React.FC<DropdownProps> = ({ title, children }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div
+      className={`mb-3 rounded-lg border-b border-gray-200 bg-white p-6 text-base dark:bg-theme-800`}
+      style={{ width: "100%" }}
```

Comment:
Not that I particularly care, but why are you using both tw classes and manual styles? Comment relevant in most of this PR

Reply:
correct, updated